Cargo.toml

```diff
@@ -11,9 +11,9 @@ members = [
     "crates/did-utils",
     "crates/keystore",
     "crates/plugin-api",
-    "crates/plugins/did-endpoint",
-    "crates/plugins/mediator-coordination",
-    "crates/plugins/oob-messages",
+    "crates/web-plugins/did-endpoint",
```

Comment:
Could not remember if this plugin has defined route(s) or not. Can you check it? Under the web module.